### DISCUSSION TITLE
Auto commit on close possible?

### DISCUSSION THREAD

**Discussion by revolvermann76** [Vinzent03/obsidian-git#351] [General]
Hey, I like Obsidian Git. Thank you for this nice Plugin. 

Normally, I set the configuration, to initiate a commit and push every 5 Minutes. This interval is okay, but sometimes I make a change to a document and close the app. The last changes are not committed then. Is it possible, to create an option to initiate a commit/push whenever the close-Event of the Obsidian-App fires?

**Comment by Vinzent03** [comment]
Thanks. This was already discussed in #13
